__tests__/testResultProcessor.test.js

```diff
@@ -41,4 +45,21 @@ describe('jest-junit', () => {
     const xmlDoc = libxmljs.parseXml(fs.writeFileSync.mock.calls[0][1]);
     expect(xmlDoc).toBeTruthy();
   });
+
+  it('should remove invalid xml characters', () => {
+    const failingTestInvalidXMLCharacter = require('../__mocks__/failing-test-invalid-xml-character.json');
+    testResultProcessor(failingTestInvalidXMLCharacter);
+
+    // Ensure fs.writeFileSync is called
+    expect(fs.writeFileSync).toHaveBeenCalledTimes(1);
+
+    // Ensure file would have been generated
+    expect(fs.writeFileSync).toHaveBeenLastCalledWith(path.resolve('junit.xml'), expect.any(String));
+
+    console.log(fs.writeFileSync.mock.calls[0][1]);
+
+    // Ensure generated file is valid xml
+    const xmlDoc = libxmljs.parseXml(fs.writeFileSync.mock.calls[0][1]);
+    expect(xmlDoc).toBeTruthy();
```

Comment:
This test doesn't actually confirm that xml 1.0 invalid characters aren't present

Reply:
It doesn't directly confirm but if you remove `removeXMLInvalidChars` from [here](https://github.com/jest-community/jest-junit/pull/94/files#diff-de6bb0568cec60450b473561b52bbf0dR183) the xml file will fail to validate.